docs/source/design.rst

```diff
@@ -71,17 +71,17 @@ A pseudopotential family can be constructed manually, by first constructing the
     from aiida import plugins
 
     UpfData = plugins.DataFactory('pseudo.upf')
-    PseudoPotentialFamily = plugins.GroupFactory('pseudo')
+    PseudoPotentialFamily = plugins.GroupFactory(entry_point_name='pseudo.family')
```

Comment:
Don't think this change is necessary, is it?
```suggestion
    PseudoPotentialFamily = plugins.GroupFactory('pseudo.family')
```

Reply:
Yeah, I just put it to remember what it is, we can leave it as it is.